submissions/Vlada-khom/html-css-popup/styles.css

```diff
@@ -0,0 +1,185 @@
+body {
+  background-color: rgba(0, 255, 255, 0.2);
+}
+
+header nav {
```

Comment:
Avoid styling by HTML tags. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Also, avoid creating too specific CSS selectors. [Read more about it](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

Reply:
Not fixed, try to use as shorter CSS selectors as you can, and avoid unnecessary nesting in scss.